skyvern/webeye/actions/actions.py

```diff
@@ -36,6 +36,7 @@ class Action(BaseModel):
 
 
 class WebAction(Action, abc.ABC):
+    frame: str | int
```

Comment:
Same here, I think we only need to update the action handling logic. Specifically how playwright interacts with the page. 

We can build and store the `element_id_to_frame` mapping in the `ScrapedPage` object we build as a result of `scrape_web_unsafe`.

Then, we can pass that info to `resolve_locator` and trace back to the original frame there without changing the way LLM, Action objects work.

WDYT?

Reply:
Sounds good! I have just pushed the update with id -> frame mapping. Please let me know, if everything is in order.